src/pretix/plugins/paypal/views.py

```diff
@@ -175,6 +175,15 @@ def webhook(request, *args, **kwargs):
     event_body = request.body.decode('utf-8').strip()
     event_json = json.loads(event_body)
 
+    # V1/V2 Sorting -- Start
+    if 'event_type' not in event_json:
+        return HttpResponse("Invalid body, no event_type given", status=400)
+
+    if event_json['event_type'].startswith(('CHECKOUT.ORDER.', 'PAYMENT.CAPTURE.')):
+        from pretix.plugins.paypal2.views import webhook
+        return webhook(request, *args, **kwargs)
+    # V1/V2 Sorting -- End
```

Comment:
Is there any way this is ever called, since paypal.views.webhook is only ever called from paypal2.views.webhook?